contracts/contracts/layer_0/clearing/ClearingStorageWrapper2.sol

```diff
@@ -837,6 +866,27 @@ abstract contract ClearingStorageWrapper2 is
         ].add(_clearingId);
     }
 
+    function _restoreAllowance(
+        address _tokenHolder,
+        bytes32 _partition,
+        IClearing.ClearingOperationType _clearingOperationType,
+        uint256 _clearingId,
+        uint256 _amount
+    ) internal {
```

Comment:
Create in ERC20StorageWrapper2 a new method called _increaseAllowedBalance and remove _restoreAllowance.